components/brave_shields/browser/domain_block_navigation_throttle.cc

```diff
@@ -97,9 +101,22 @@ DomainBlockNavigationThrottle::WillStartRequest() {
   if (!request_url.SchemeIsHTTPOrHTTPS())
     return content::NavigationThrottle::PROCEED;
 
+  // Only block if Brave Shields is enabled and tracker/ad blocking is set to
+  // aggressive.
+  content::WebContents* web_contents = handle->GetWebContents();
+  HostContentSettingsMap* host_content_settings_map =
+      HostContentSettingsMapFactory::GetForProfile(
+          Profile::FromBrowserContext(web_contents->GetBrowserContext()));
+  if (!brave_shields::GetBraveShieldsEnabled(host_content_settings_map,
+                                             request_url))
+    return content::NavigationThrottle::PROCEED;
+  LOG(ERROR) << brave_shields::GetAdControlType(host_content_settings_map, request_url);
```

Comment:
You left a `LOG` here :smiley:

Reply:
removed in https://github.com/brave/brave-core/commit/7ea545515d48b6a3678cbd1244d568a312dc6826